dane_workflows/status_monitor.py

```diff
@@ -24,6 +25,9 @@ def __init__(self, config: dict, status_handler: StatusHandler):
             if "CONFIG" in config["STATUS_MONITOR"]
             else {}
         )
+        # add PROC_ENV.CONFIG and EXPORTER.CONFIG as they contain relevant values for monitoring
+        self.config["PROC_ENV"] = config["PROC_ENV"]
```

Comment:
this is not very nice. Please assign these configs to separate variables e.g. self.proc_env_conf and self.exporter_conf

also you should not validate parts that are not specific for the status_monitor config, since they are validated by their own components.

Reply:
@Jaap resolved by last push: [d74987f](https://github.com/beeldengeluid/dane-workflows/pull/15/commits/d74987f9d612e3bed10c5fe9753ff01607544776), please review

Reply:
ok nice. Will also test it tomorrow, so we can merge this one and move on :-)